### DISCUSSION TITLE
Can't Showing Full Image in The 2K Resolution

### DISCUSSION THREAD

**Discussion by vasluex** [t1m0thyj/WinDynamicDesktop#317] [Q&A]
<img src="https://cdnimg.acgget.com/images/202105/187a90fd40b744c4.jpg" />
Can it be fixed？

**Comment by t1m0thyj** [comment]
The default behavior of Windows for JPEG wallpapers is to compress them which reduces the image quality. It may help to use the script [here](https://github.com/t1m0thyj/WDD-scripts/tree/master/registry#disableenable-jpeg-compression) to disable JPEG compression.
